caom2-tap-server/build.gradle

```diff
@@ -32,9 +32,10 @@ dependencies {
     compile 'org.opencadc:cadc-tap-schema:[1.1.6,)'
     compile 'org.opencadc:cadc-adql:[1.1,)'
     
-    compile 'org.opencadc:cadc-vos:1.+'
-    compile 'org.opencadc:cadc-access-control:[1.1.1,)'
-    compile 'org.opencadc:cadc-access-control-identity:1.+'
+    compile 'org.opencadc:cadc-vos:[1.0,)'
+    compile 'org.opencadc:cadc-gms:[1.0.0,)'
+    compile 'org.opencadc:cadc-access-control:[1.1.1,2.0)'
+    compile 'org.opencadc:cadc-access-control-identity:[1.0,2.0)'
```

Comment:
compile time dependency on cadc-access-control-identity is probably a mistake from when we moved this code from service.git/tap into a library... should be removed.

Reply:
done